packages/core/test/todo.md

```diff
@@ -0,0 +1,2024 @@
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         key: 'a',
+//     }],
+//     test: "<span><b>dom<br/>a&nbsp;to edit</b></span>",
+//         start: "b:contents()[2]->1",
+//     },
+// },
+// {
+//     name: "in span > b: SHIFT+ENTER -> ENTER -> 'a'",
+//     content: "<span><b>dom to edit</b></span>",
+//     steps: [{
+//         start: "b:contents()[0]->3",
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: "<span><b>dom<br/></b></span><br/><span><b>a&nbsp;to edit</b></span>",
+//         start: "b:eq(1):contents()[0]->1",
+//     },
+// },
+// {
+//     name: "in p: 2x SHIFT+ENTER -> 'a'",
+//     content: "<p>dom to edit</p>",
+//     steps: [{
+//         start: "p:contents()[0]->3",
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         key: 'a',
+//     }],
+//     test: "<p>dom<br/><br/>a&nbsp;to edit</p>",
+//         start: "p:contents()[3]->1",
+//     },
+// },
+// {
+//     name: "in p: ENTER -> SHIFT+ENTER -> 'a'",
+//     content: "<p>dom to edit</p>",
+//     steps: [{
+//         start: "p:contents()[0]->3",
+//         key: 'ENTER',
+//     }, {
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         key: 'a',
+//     }],
+//     test: "<p>dom</p><p><br/>a&nbsp;to edit</p>",
+//         start: "p:eq(1):contents()[1]->1",
+//     },
+// },
+// {
+//     name: "in empty-p (p before and after): ENTER -> 'a'",
+//     content: "<p>dom </p><p><br/></p><p>to edit</p>",
+//     steps: [{
+//         start: "p:eq(1):contents()[0]->0",
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: "<p>dom </p><p><br/></p><p>a</p><p>to edit</p>",
+//         start: "p:eq(2):contents()[0]->1",
+//     },
+// },
+// {
+//     name: "in p: SHIFT+ENTER at end -> '寺'",
+//     content: "<p>dom </p><p>to edit</p>",
+//     steps: [{
+//         start: "p:first:contents()[0]->4",
+//         key: 'ENTER',
+//         shiftKey: true,
+//     }, {
+//         keyCode: 23546, /*temple in chinese*/
+//     }],
+//     test: "<p>dom <br/>寺</p><p>to edit</p>",
+//         start: "p:first:contents()[2]->1",
+//     },
+// },
+// {
+//     name: "in empty-p (div > a after): 3x SHIFT+ENTER -> 'a'",
+//     content: "<p><br/></p><div><a href='#'>dom to edit</a></div>",
+//     steps: [{
+//             start: "p->1",
+//             key: 'ENTER',
+//             shiftKey: true,
+//         },
+//         {
+//             key: 'ENTER',
+//             shiftKey: true,
+//         },
+//         {
+//             key: 'ENTER',
+//             shiftKey: true,
+//         },
+//         {
+//             key: 'a',
+//         }
+//     ],
+//     test: "<p><br/><br/><br/>a</p><div><a href=\"#\">dom to edit</a></div>",
+//         start: "p:contents()[3]->1",
+//     },
+// },
+```
```

Comment:
Add:
```markdown
## List (`toggleList`)<a name="list"></a>

### To convert later<a name="l-todo"></a>
(when their related feature is implemented)

#### Unbreakable<a name="l-t-unbreakable"></a>
```typescript
it('should turn a paragraph in a div and a paragraph into a list with two items', async () => {
    await testEditor({
        contentBefore: '<div><p>ab[cd</p></div><p>ef]gh</p>',
        stepFunction: toggleUnorderedList,
        contentAfter: '<div><ul><li>ab[cd</li></ul></div><ul><li>ef]gh</li></ul>',
    });
});
```
```